libs/lb/lb-rs/src/service/account_service.rs

```diff
@@ -47,7 +49,14 @@ impl<Client: Requester, Docs: DocumentService> CoreState<Client, Docs> {
         Ok(account)
     }
 
-    pub(crate) fn import_account(&mut self, account_string: &str) -> LbResult<Account> {
+    pub(crate) fn import_account(&mut self, key: AccountKey) -> LbResult<Account> {
```

Comment:
I think you misunderstood my intention. Presently the UI dev still has to try to determine if the thing the user put into the box is base64 or a 12 word phrase. 

I want core to just take a string and completely abstract that concept away from the consumer of core